mcgeneration/helpers/Gridpack.py

```diff
@@ -62,7 +62,7 @@ def __init__(self,**kwargs):
             'save_diagrams': False,         # Runs a modified version of the generation script that exits early to keep feynman diagrams
             'use_coupling_model': False,    # Use the 'coupling_orders' version of the dim6 model
             'coupling_string': None,        # If not None replaces "DIM6=1" with the specified string in the process card
-            'replace_model': None,          # If not None overwrites the import model line of the process card
+            'replace_model': None,          # If not None needs to be a tuple/list of length 2 that specifies the old and new model to be replaced
```

Comment:
Can you put an example in the README just to be super clear? I'm guessing it's just
```python
replace_model=('dim6top', 'dim6top_extra')
```

Reply:
Done